authoidc.properties

```diff
@@ -0,0 +1,23 @@
+category=Integration
+description=Delegate authentication using the OpenID Connect protocol
+homepageUrl=https://github.com/sonar-auth-oidc/sonar-auth-oidc
+archivedVersions=
+publicVersions=2.1.1,3.0.0
+
+defaults.mavenGroupId=org.vaulttec.sonarqube.auth.oidc
+defaults.mavenArtifactId=sonar-auth-oidc-plugin
+
+3.0.0.description=Add compatibility with Sonarqube v25.x/2025.x.
+3.0.0.sqs=[2025.1,2025.2,LATEST]
+3.0.0.sqcb=[25.1,25.2,LATEST]
```

Comment:
Same here. Start with 25.1 (should probably actually be 24.12) and end with LATEST. No middle value.